src/creatures/players/player.h

```diff
@@ -2466,6 +2466,7 @@ class Player final : public Creature, public Cylinder, public Bankable {
 		 */
 		std::vector<Item*> getEquippedItems() const;
 
+
```

Comment:
For good practice, you can delete empty lines and spaces.

Reply:
Fixed! :)